### PR TITLE
chore/makePointEditorLookGood

### PR DESCRIPTION
**Pull request by Luke-Clements**
# Description

UI updates.

## Type of change

UI

# How Has This Been Tested?

Manually on localhost

### CODE REVIEW

**Review by alfredovargasjr — APPROVED** [review]

**Comment by Luke-Clements** [comment]
Don't merge it! There is one thing still not working properly.

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]
Not ready

**Review by alfredovargasjr — APPROVED** [review]

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]
Adding notifcations

**Review by alfredovargasjr — APPROVED** [review]

File: Frontend/src/views/PointEditor.vue [hunk 1]
```diff
@@ -1,54 +1,68 @@
...
-    <div>
-      <div id="map"></div> 
-    </div>
-  </div>
+          </v-form>
+          <div v-if="loading">
+            <Loading :dialog="loading" :text="loadingText"/>
+          </div>
+          <div v-if="notification">
+            <v-snackbar
```
Comment (alfredovargasjr) [inline Frontend/src/views/PointEditor.vue:47]: Can you make this a color=success

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

**Review by alfredovargasjr — APPROVED** [review]

**Pull request merged**